### DISCUSSION TITLE
How to disable a channel?

### DISCUSSION THREAD

**Discussion by DerEwige** [ACINQ/eclair#2477] [Node operators]
I have the problem that many of my mutual close attempts with LND nodes lead to force closes.
This is what happens:

I request a mutual close.
LND checks if the channel is clear.
Channel has pending updates/htlcs.
LND sends back an error stating that the channel can not be closed right now.
Eclair immediately force closes the channel as a result of that error.

To prevent this from happening, I would like to disable the channel on my side and wait a few seconds before trying a mutual close, but I do not see any option for this in the API

**Comment by t-bast** [comment]
I have heard people complain about this before, I was hoping lnd had fixed it...this is an lnd issue, they have no good reason for sending an `error` message here. The `shutdown` message is explicitly defined as a signal that we will initiate a channel close and can perfectly be sent while the channel has pending updates/htlcs. After exchanging `shutdown` both peers will wait for the htlcs to clear.

I don't see how we could work around this without a bunch of very annoying additional state tracking in `eclair`, which I'm very reluctant to add to work around another implementation misbehaving. There is a one-line fix on their side that would make this go away: send a `warning` instead of an `error`. Can you open an issue on their repository and suggest this trivial fix?

**Reply by sstone** [reply]
I think there's already an issue for this: https://github.com/lightningnetwork/lnd/issues/6039

**Reply by DerEwige** [reply]
you are right. just checked all the other linked issues
[https://githhttps://github.com/ElementsProject/lightning/issues/5295ub.com/ElementsProject/lightning/issues/5295](https://github.com/lightningnetwork/lnd/issues/6599)
This exactly is the situation I have like every 3rd close

So yes there is already an open issue
